catalog/spatial/geocoding/spatial-geocoding-create/src/main/java/org/codice/ddf/spatial/geocoding/create/GeoNamesCreator.java

```diff
@@ -43,13 +36,14 @@ public GeoEntry createGeoEntry(final String line, String entryResource) {
     // Passing a negative value to preserve empty fields.
     final String[] fields = line.split("\\t", -1);
 
-    String countryCode = fields[8];
-    Set<CountryCode> countryCodes =
-        converter.fromAlpha2(
-            fields[8].trim(), isoStandard.getStandard(), isoStandard.getStandard());
-    if (CollectionUtils.isNotEmpty(countryCodes)) {
-      countryCode = countryCodes.iterator().next().getAsFormat("alpha3");
-    }
+    String countryCode =
+        isoStandard
+            .getStandardEntries()
+            .stream()
+            .filter(c -> c.getAsFormat("alpha2").equals(fields[8]))
```

Comment:
✏️ Could we use an enumeration/static constant to describe what the magic index 8 does?  I do see there's lots of these down on line 49-54 as well.

Reply:
I updated the source to indicate that fields[8] is an alpha 2 country code. I think the the rest of the indexes are clear from context.